src/App/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css } from 'emotion';
+import colors from '../utils/styles/colors';
+
+export const alertError = css`
+  background-color: ${colors.red} !important;
+  min-width: inherit !important;
+`;
+
+export const alertErrorMsg = css`
+  display: flex;
```

Comment:
I have seen that in some places you are using template literals and in other places an object ... I think we should decide which one to use and follow a standard

Reply:
Glad you raise this point :)
I don't have a strong opinion on this.
Template literals feels more natural to me because it is closer to scss with simple styles. Object may be more optimal? when parsing a function.
Thoughts?

Reply:
Hi.Thank you for asking..I would rather use objects ... since we can benefit from autocompletion and since we plan to migrate to material-ui/styles. 😉

Reply:
Objects I think would be the way (not 100% sure), but for this PR template literals are more than enough, we use them all over the place.

Reply:
I have updated the simple ones below.
Perhaps a slow transition as more and more Material UI components gets added?

Reply:
fyi, have reverted again. I can't find a way to remove the compilation error it throws for some odd reasons if all exports are css objects
```
[0] src/components/Login/styles.ts
[0]  1:1  ✖  Unexpected empty source   no-empty-source
```

Reply:
@griffithtp switch `no-empty-source` to warning for now in the `.eslintrc` folder, we will investigate that later on.